rca/programmes/models.py

```diff
@@ -596,16 +597,21 @@ class ProgrammePage(BasePage):
         ),
     ]
     api_fields = [
-        APIField("degree_level", serializer=degree_level_serializer()),
+        # Fields for filtering and display, shared with shortcourses.ShortCoursePage.
         APIField("subjects"),
         APIField("programme_type"),
-        APIField("programme_description_subtitle"),
-        APIField("pathway_blocks"),
+        APIField("related_schools_and_research_pages"),
+        APIField(
+            "summary",
+            serializer=CharFieldSerializer(source="programme_description_subtitle"),
```

Comment:
@kevinhowbrook FYI – renaming this field could be problematic if there is other code using the API (I’ve assumed there wasn’t).

Reply:
Noted, thanks. At the moment there isn't but I also don't expect this to change it. I imagine if we need to we can expose a custom field to the API though to dodge any issues